### PR TITLE
Add com.github.rkoesters.xkcd-gtk

### PR DESCRIPTION
**Pull request by rkoesters**
This is an app I made and distributed via elementaryOS's AppCenter. [A user requested I create a flatpak version](https://github.com/rkoesters/xkcd-gtk/issues/40).

### CODE REVIEW

File: com.github.rkoesters.xkcd-gtk.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
+{
+	"app-id": "com.github.rkoesters.xkcd-gtk",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "1.6",
```
Comment (barthalion) [inline com.github.rkoesters.xkcd-gtk.json]:
```suggestion
	"runtime-version": "18.08",
```

File: com.github.rkoesters.xkcd-gtk.json [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+				{
+					"type": "git",
+					"url": "https://github.com/rkoesters/xkcd-gtk.git",
+					"tag": "1.0.0",
+					"disable-shallow-clone": true
+				},
+				{
+					"type": "archive",
+					"dest": "goroot",
+					"url": "https://dl.google.com/go/go1.10.4.linux-amd64.tar.gz",
```
Comment (barthalion) [inline com.github.rkoesters.xkcd-gtk.json]: There is golang SDK extension available. Please use it instead.

File: com.github.rkoesters.xkcd-gtk.json [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+					"sha256": "fa04efdb17a275a0c6e137f969a1c4eb878939e91e1da16060ce42f02c2ec5ec"
+				},
+				{
+					"type": "git",
+					"dest": "src/github.com/RoaringBitmap/roaring",
+					"url": "https://github.com/RoaringBitmap/roaring.git",
+					"branch": "master"
```
Comment (barthalion) [inline com.github.rkoesters.xkcd-gtk.json]: All git sources should point to specific commits or tags.

File: com.github.rkoesters.xkcd-gtk.json [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+	"runtime-version": "1.6",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "com.github.rkoesters.xkcd-gtk",
+	"finish-args": [
+		"--socket=x11",
+		"--socket=wayland",
+		"--share=network"
+	],
+	"cleanup": [
+		"/lib/debug"
```
Comment (barthalion) [inline com.github.rkoesters.xkcd-gtk.json]: Don't cleanup any debug data, it is extracted to .Debug extension.

**Comment by nedrichards** [comment]
bot, build com.github.rkoesters.xkcd-gtk

**Comment by nedrichards** [comment]
bot, build com.github.rkoesters.xkcd-gtk

**Comment by rkoesters** [comment]
> Does the application look fine without the elementary gtk & icons themes? If not, you should use the elementary baseApp & patch the application to force using it. Otherwise, the application won't look that good...

It looks perfectly fine without the elementary themes. I have tested it with Adwaita, Ambiance, and Mint-Y.

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/com.github.rkoesters.xkcd-gtk

**Pull request closed without merging**